packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -49,6 +51,18 @@ function addVNodeToChildLWC(vnode: VCustomElement) {
     ArrayPush.call(getVMBeingRendered()!.velements, vnode);
 }
 
+// [st]atic node
+function st(elmProto: Element, key: Key): VStatic {
+    return {
+        type: VNodeType.Static,
+        sel: undefined,
```

Comment:
any particular reason for this to not have a sel even though I understand it is not going to be used?

Reply:
yes, it is not used, and we do not know the selector (without processing the string).